### PR TITLE
Add escaping of the test names for passing as command line parameters

### PR DESCRIPTION
**Pull request by kant2002**
See #230 for details.

### CODE REVIEW

**Comment by kant2002** [comment]
Please @mousetraps look at this too.

File: Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs [hunk 1]
```diff
@@ -111,6 +111,11 @@ class TestFramework {
             return path;
         }
 
+        private string WrapTestNameWithQuot(string path)
```
Comment (mousetraps) [inline Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs]: Minor spelling/grammar error - WrapTestNameWithQuotes 

We should also correct the other method name as well while we're at it

File: Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs [hunk 2]
```diff
@@ -111,6 +111,11 @@ class TestFramework {
...
+        {
```
Comment (mousetraps) [inline Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs]: Brace on same line

File: Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs (hunk 1 above)
Comment (mousetraps) [inline Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs]: Also the name of the argument going in should just be testName rather than path.

Comment (mousetraps) [inline Nodejs/Product/TestAdapter/TestFrameworks/TestFramework.cs]: Lastly including this near the WrapWithQuotes method will make the code a tad more organized

**Comment by mousetraps** [comment]
Minor nitpicky comments and lgtm after that!

**Comment by kant2002** [comment]
I made change @mousetraps. Please review.

**Comment by mousetraps** [comment]
Awesome Thx and merged!

**Pull request merged**